script/gitlabci/job_base.yml

```diff
@@ -1,40 +1,48 @@
 .base:
-  image: registry.gitlab.com/hzdr/crp/alpaka-group-container/alpaka-ci-${ALPAKA_CONTAINER_SUFFIX}:${ALPAKA_GITLAB_CI_CONTAINER_VERSION}
   variables:
     ALPAKA_DEBUG: 0
-    BOOST_ROOT: /opt/boost/${ALPAKA_BOOST_VERSION}
-    BOOST_LIBRARYDIR: /opt/boost/${ALPAKA_BOOST_VERSION}/lib
-    ALPAKA_CI_BOOST_LIB_DIR: /opt/boost/${ALPAKA_BOOST_VERSION}
+    ALPAKA_CI_BOOST_BRANCH: "boost-${ALPAKA_BOOST_VERSION}"
+    BOOST_LIBRARYDIR: "/opt/boost/${ALPAKA_BOOST_VERSION}/lib"
+    ALPAKA_CI_OS_NAME: "Linux"
   script:
-    # create fake sudo
-    - cp ./script/gitlabci/fake_sudo.sh /bin/sudo
-    - chmod +x /bin/sudo
-    - mkdir -p ${ALPAKA_CI_CMAKE_DIR}
-    - source ./script/install_cmake.sh
+    - source ./script/before_install.sh
+    - source ./script/install.sh
     - source ./script/run.sh
+  interruptible: true
 
-.base_gcc:
+.base_cuda_gcc:
+  image: nvidia/cuda:${ALPAKA_CI_CUDA_VERSION}-devel-ubuntu${ALPAKA_CI_UBUNTU_VER}
```

Comment:
What is wrong with the alpaka container? Can you please open an issue, that it can be fixed: https://gitlab.com/hzdr/crp/alpaka-group-container

Reply:
offline discussed: https://github.com/alpaka-group/alpaka/pull/1407#issue-727718589 clang is using gcc 5 header and therefore compiling with clang cuda is not possible. 
Maybe we need to build a new alpaka container version. Integration of alpaka container will be the next step.
This PR is for using the full existing alpaka toolchain.